easybuild/easyconfigs/h/HOLE2/HOLE2-2.3.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'HOLE2'
+version = '2.3.1'
+
+homepage = 'https://www.holeprogram.org/'
+description = """HOLE is a program that allows the analysis and visualisation of
+the pore dimensions of the holes through molecular structures of ion channels"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/osmart/hole2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['1109dd3d15a63b6c72833314f3ef0fcfdf56341e634fbd2195df7427e6b435ae']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+]
+
+skipsteps = ['configure']
+
+prebuildopts = "".join([
+    "source source.apache && ",
+    "cd src && ",
+])
+
+preinstallopts = "source source.apache && cd src && "
+installopts = "PREFIX=%(installdir)s"
+postinstallcmds = [
+    # additional install targets: install-opt and install-2dmap
+    "source source.apache && cd src && make PREFIX=%(installdir)s install-all",
+]
```

Comment:
Kind of repeating things here, think we can simplify this to
```
prebuildopts = "source source.apache && cd src && "
preinstallopts = prebuildopts
install_cmd = "make install-all"
```

Reply:
Oh I didn't know `install_cmd`. Thanks!